.vscode/launch.json

```diff
@@ -4,7 +4,14 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
-
+    {
```

Comment:
Does this file really belong on github? isn't this rather a personal ide configuration file?